universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java

```diff
@@ -24,17 +27,62 @@
   private QuestionService questionService;
   private QuestionDefinition questionDefinition;
   private ProgramDefinition programDefinition;
+  private ApplicantRepository applicantRepository;
 
   @Before
   public void setUp() throws Exception {
     subject = instanceOf(ApplicantServiceImpl.class);
     programService = instanceOf(ProgramServiceImpl.class);
     questionService = instanceOf(QuestionService.class);
-
+    applicantRepository = instanceOf(ApplicantRepository.class);
     createQuestions();
     createProgram();
   }
 
+  @Test
```

Comment:
suggestion (blocking): We should add a test for an error case - perhaps "Block not found"?

Reply:
done